daemon/containerd/image_list.go

```diff
@@ -276,10 +279,15 @@ func (i *ImageService) singlePlatformImage(ctx context.Context, contentStore con
 		return nil, nil, err
 	}
 
+	created := int64(0)
+	if cfg.Created != nil {
+		created = cfg.Created.Unix()
```

Comment:
Micro-nit; as we already have a `summary` var in which we store the image-data, I guess we can avoid the `created` var if we put this block after it;


```go
if cfg.Created != nil {
    summary.Created = cfg.Created.Unix()
}
```

Would there be a need to do ` && !cfg.Created.IsZero()` as well? (not sure)

Reply:
I think it's fine to set the `Created` in the `IsZero` case, that's actually what the GD case does

Reply:
Oh! We can; but that's fine for a separate PR, as that's not a new change, right?

My comment was mostly; if we want to; let's do so before merging, to reduce (ever so little) code churn 👍